src/duckstation-qt/memorycardeditordialog.cpp

```diff
@@ -30,6 +30,13 @@ MemoryCardEditorDialog::MemoryCardEditorDialog(QWidget* parent) : QDialog(parent
   connectUi();
   populateComboBox(m_ui.cardAPath);
   populateComboBox(m_ui.cardBPath);
+
+  static constexpr char new_card_hover_text[] = TRANSLATABLE("MemoryCardEditorDialog", "New Card...");
```

Comment:
This can just be `m_ui.newCardA->setTooltip(tr("New Card..."));`

Reply:
Whoops, not sure why I overcomplicated that... thanks for fixing it.